AntistasiOfficial.Altis/resourcecheck.sqf

```diff
@@ -154,13 +154,19 @@ while {true} do {
 	} forEach ciudades;
 	
 	_types = ["FOOD", "WATER", "FUEL"]
+	//@Stef i have reduced the amount to maximum 2 missions per tick each with 85% chance
```

Comment:
Basically on resourcecheck 2 or 3 towns should have their tier degraded by 1 level... mission is activated in another moment, so when you find certain supplies and decide where to deliver